iocore/net/P_Net.h

```diff
@@ -104,7 +104,6 @@ extern RecRawStatBlock *net_rsb;
 #include "P_UnixPollDescriptor.h"
 #include "P_Socks.h"
 #include "P_CompletionUtil.h"
-#include "P_NetVCTest.h"
```

Comment:
Had to correct many improper dependencies on the fact that P_NetVCTest.h is included here and included Regression.h.